student.rb

```diff
@@ -1,17 +1,14 @@
-class Student
-  attr_reader :age, :phase
-  attr_accessor :name
+require_relative 'person'
+require_relative 'high_five'
```

Comment:
Yes, you must require the file to use a module within a class, but you _also_ have to include the module.